packages/calcite-components/src/components/fab/fab.scss

```diff
@@ -1,17 +1,33 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-fab-shadow: Specifies the shadow of the component.
+ * @prop --calcite-fab-shadow-hover: Specifies the shadow of the component when in hover state.
+ * @prop --calcite-fab-shadow-active: Specifies the shadow of the component when in active state.
+ */
+
 :host {
-  @apply flex bg-transparent;
+  @apply flex;
+  --calcite-fab-shadow: 0 6px 20px -4px rgba(0, 0, 0, 0.1), 0 4px 12px -2px rgba(0, 0, 0, 0.08);
+  --calcite-fab-shadow-active: 0 2px 12px -4px rgba(0, 0, 0, 0.2), 0 2px 4px -2px rgba(0, 0, 0, 0.16);
+  --calcite-fab-shadow-hover: var(--calcite-shadow-md);
+  --calcite-fab-offset-shadow: 0 0 #0000;
+  background-color: var(--calcite-color-transparent);
 }
 
-@include disabled();
-
 calcite-button {
-  @apply shadow-2;
+  box-shadow: var(--calcite-fab-offset-shadow), var(--calcite-fab-offset-shadow), var(--calcite-fab-shadow);
```

Comment:
☝️ There are 3 different css custom properties used in the `shadow-2` tailwind util that don't seem to be completely replicated here.

![image](https://github.com/Esri/calcite-design-system/assets/821864/681437e1-dff3-4ab7-ba72-38d727b78dab)

Reply:
So I was confused about the shadow-2, so I asked Ali for suggestions and here's two suggestions they gave

```
:host {
  @apply flex;
  
  --calcite-fab-shadow: 0 6px 20px -4px rgba(0, 0, 0, 0.1), 0 4px 12px -2px rgba(0, 0, 0, 0.08);
  --calcite-fab-shadow-active: 0 2px 12px -4px rgba(0, 0, 0, 0.2), 0 2px 4px -2px rgba(0, 0, 0, 0.16);
  --calcite-fab-shadow-hover: var(--calcite-shadow-md);

  --calcite-fab-offset-shadow:  0 0 #0000;

  --calcite-button-box-shadow: var(--calcite-fab-offset-shadow), var(--calcite-fab-offset-shadow), var(--calcite-fab-shadow);

  background-color: var(--calcite-color-transparent);
  &:hover {
    --calcite-button-box-shadow: var(--calcite-fab-offset-shadow), var(--calcite-fab-offset-shadow), var(--calcite-fab-shadow-hover);
  }
  &:active {
    --calcite-button-box-shadow: var(--calcite-fab-offset-shadow), var(--calcite-fab-offset-shadow), var(--calcite-fab-shadow-active);
  }
}

@include disabled();

@include base-component();
```

and 


```
:host {
  @apply flex;
  
  --calcite-fab-shadow: 0 6px 20px -4px rgba(0, 0, 0, 0.1), 0 4px 12px -2px rgba(0, 0, 0, 0.08);
  --calcite-button-box-shadow: var(--calcite-internal-fab-offset-shadow), var(--calcite-internal-fab-offset-shadow), var(--calcite-fab-shadow);
  
  --calcite-internal-fab-offset-shadow:  0 0 #0000;

  background-color: var(--calcite-color-transparent);
  
  &:hover {
      --calcite-fab-shadow: var(--calcite-shadow-md);
  }
  
  &:active {
    --calcite-fab-shadow: 0 2px 12px -4px rgba(0, 0, 0, 0.2), 0 2px 4px -2px rgba(0, 0, 0, 0.16);
  }
}

@include disabled();

@include base-component();
```
The problem I had with those suggestions is that they were not actually applying the shadows because the shadows exists on the `calcite-button` within the fab. So I made some tweaks to the suggestion and visually inspected the component to see if it looked the same as before adding the tokens. It did look similar but I also wanted to see if there would be any changes caught by chromatic.

Reply:
Is it possible to use tokens to set shadow on the internally rendered Button vs. applying them on the FAB itself? Would require Button to land first.

Reply:
cc @alisonailea for above

Reply:
Token must be refactored to only set it's tokens(css vars) on :host. I'll have a PR up today with a button refactor that fixes the problem with passing tokens down. In the case of `fab` and other components intended to wrap specific calcite-components. I think the best pattern would be to simply document the component tokens for the wrapped component in the docs for this component. Because of the way CSS variables pierce the shadowdom any CSS tokens set in the `:host` of one component will be lifted up into it's parent `:host` when wrapped within the shadow dom of another component.

Reply:
Sorry @DitwanP. I know this is different than the code samples I sent you initially.